collect_app/src/main/res/xml/admin_preferences.xml

```diff
@@ -18,6 +18,13 @@
         app:allowDividerAbove="false"
         app:allowDividerBelow="false"
         app:iconSpaceReserved="false">
+
+        <EditTextPreference
+            android:key="project_name"
```

Comment:
I'm thinking this should probably have `app:persistent="false"` (like we do with Device ID in `form_metadata_preferences.xml`).

Reply:
Done.